src/core/js/models/adaptModel.js

```diff
@@ -55,15 +55,17 @@ define(function (require) {
         },
 
         setupChildListeners: function() {
+            var children = this.getChildren();
 
-            if (!this.getChildren()) return;
-
-            this.listenTo(Adapt[this._children], {
-                "change:_isReady": this.checkReadyStatus,
-                "change:_isComplete": this.onIsComplete,
-                "change:_isInteractionComplete": this.checkInteractionCompletionStatus
-            });
+            if (!children) return;
 
+            children.each(function(child) {
```

Comment:
Can you add this in? 
```javascript
                this.stopListening(child, {
                    "change:_isReady": this.checkReadyStatus,
                    "change:_isComplete": this.onIsComplete,
                    "change:_isInteractionComplete": this.checkInteractionCompletionStatus
                });
```
So that if we change the children we can get the model to re-listen properly?